modules/rosa-sts-byo-oidc.adoc

```diff
@@ -1,29 +1,36 @@
 // Module included in the following assemblies:
 //
-// * rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly.adoc
+// * rosa_architecture/rosa-oidc-overview.adoc
 // * rosa_architecture/rosa-sts-about-iam-resources.adoc
+// * rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly.adoc
+// * rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-quickly.adoc
+// * rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-with-customizations.adoc
 
 ifeval::["{context}" == "rosa-hcp-sts-creating-a-cluster-quickly"]
 :rosa-hcp:
 endif::[]
-ifeval::["{context}" == "rosa-sts-about-iam-resources"]
-:rosa-classic:
-endif::[]
 
 :_content-type: PROCEDURE
-[id="rosa-hcp-byo-oidc_{context}"]
+[id="rosa-sts-byo-oidc_{context}"]
 = Creating an OpenID Connect configuration
 
-When using a {hcp-title} cluster, you must create the OpenID Connect (OIDC) configuration prior to creating your cluster. This configuration is registered to be used with OCM.
+When using a 
+ifdef::rosa-hcp[]
+{hcp-title} cluster, you must 
+endif::rosa-hcp[]
+ifndef::rosa-hcp[]
+{product-title} cluster, you may 
+endif::rosa-hcp[]
+create the OpenID Connect (OIDC) configuration prior to creating your cluster. This configuration is registered to be used with OCM.
```

Comment:
Somewhat out of scope:

At least on the pages you provided for a preview, I see both "OpenShift Cluster Manager" and "OCM" used throughout, but never together to explicitly indicate what the abbreviation is shorthand for. If it's possible to do in this PR, or a later effort, I'd suggest adding "(OCM)" after the first instance of "OpenShift Cluster Manager" in assemblies that use this module. 

No big deal if that can't be captured in this effort.